cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -250,6 +261,7 @@ defaultPartialNodeConfiguration =
     , pncLoggingSwitch = Last $ Just True
     , pncSocketPath = mempty
     , pncDiffusionMode = Last $ Just InitiatorAndResponderDiffusionMode
+    , pncSnapshotInterval = mempty
```

Comment:
So instead of `fromMaybe DefaultSnapshotInterval $ getLast $ pncSnapshotInterval pnc` in the FromJSON instance, put `Last $ Just DefaultSnapshotInterval` here instead. This allows us to keep all our default configuration options in one place.

You can then change:
`fromMaybe DefaultSnapshotInterval $ getLast $ pncSnapshotInterval pnc`
to
`getLast $ pncSnapshotInterval pnc` 

`defaultPartialNodeConfiguration` will specify `DefaultSnapshotInterval` if nothing is specified in the node configuration yaml file.

Reference: https://medium.com/@jonathangfischoff/the-partial-options-monoid-pattern-31914a71fc67